tensorboard/components/tf_storage/storage.ts

```diff
@@ -54,6 +54,7 @@ export const {
   set: setString,
   getInitializer: getStringInitializer,
   getObserver: getStringObserver,
+  disposeBinding: disposeStringBinding,
```

Comment:
Maybe best to add these to all the premade ones in this file if we add them to some of them (i.e. also add disposeBooleanBinding and disposeObjectBinding), otherwise the inconsistency could be confusing.